geometric_kernels/kernels/feature_maps.py

```diff
@@ -123,7 +120,7 @@ def _map(X: B.Numeric, params, state, key, **kwargs) -> B.Numeric:
                  for jax, and the same random state (generator) for all other backends.
         """
         key, random_phases = space.random(key, num_random_phases)  # [O, D]
-        eigenvalues = state["eigenvalues_laplacian"]
+        eigenvalues = kernel.eigenvalues_laplacian()
```

Comment:
What about making `eigenvalues_laplacian` a property? Calling it as a method seems old-fashioned to me.